recipes/qt/6.x.x/conanfile.py

```diff
@@ -10,25 +10,31 @@
 
 
 class qt(Generator):
+    @staticmethod
+    def content_template(path, folder):
+        return textwrap.dedent("""[Paths]
+            Prefix = {0}
```

Comment:
```suggestion
        return textwrap.dedent("""\
            [Paths]
            Prefix = {0}
```
don't we need to do this,  in order to make dedent work ? (all lines need to have the same indentation)

Reply:
```
>>> textwrap.dedent("""hello
...         world""")
'hello\n        world'
```
vs 
```
>>> textwrap.dedent("""\
...     hello
...     world""")
'hello\nworld'
```
this was a mistake in `def package(self):`